modules/tensor_mechanics/test/tests/mandel_notation/finite_elastic.i

```diff
@@ -0,0 +1,112 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 3
+  ny = 3
+  nz = 3
+[]
+
+[GlobalParams]
+  displacements = 'disp_x disp_y disp_z'
+[]
+
+[Variables]
+  # scale with one over Young's modulus
+  [disp_x]
+    scaling = 1e-10
+  []
+  [disp_y]
+    scaling = 1e-10
+  []
+  [disp_z]
+    scaling = 1e-10
+  []
+[]
+
+[Kernels]
+  [stress_x]
```

Comment:
Why not use the TM/Master action?

Also, I'm looking at this input file carefully to try to figure out what the user does to trigger the use of the symmetric tensors, and I'm not seeing it. Where does the user control that?

Reply:
By choosing the symmetric stress and strain calculator classes and the symmetric stress divergence kernel. those have not yet been added to the master action

Reply:
Maybe I'm missing something here. Isn't 'ADComputeFiniteStrain' the nonsymmetric one?